src/Standards/BestIt/Sniffs/TypeHints/ReturnTypeDeclarationSniff.php

```diff
@@ -173,6 +174,18 @@ protected function fixDefaultProblem(CodeWarning $exception)
         }
     }
 
+    /**
+     * Get the value of a single PHPCS config key.
+     *
+     * @param string $key The name of the config value.
+     *
+     * @return string
+     */
+    public static function getConfigData(string $key): string
+    {
+        return Config::getConfigData($key);
```

Comment:
The config value is optional. What happens when its not declared?

Reply:
Right now, if not declared it results in an error.